SlaveWAMPServer.js

```diff
@@ -145,7 +144,14 @@ class SlaveWAMPServer extends WAMPServer {
 			callback('RPC response timeout exceeded');
 			this.deleteCall(request);
 		}, this.internalRequestsTimeoutMs);
-		return setWith(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, { callback, requestTimeout }, Object);
+
+		Object.assign(this.interProcessRPC, {
+			[request.socketId]: {
```

Comment:
Is it possible for these property names to override existing but important properties?

Reply:
I can't find any other properties that will be added or removed from `this.interProcessRPC` except for this nested structure.